lib/src/common/disposable.dart

```diff
@@ -697,7 +650,7 @@ class Disposable implements _Disposable, DisposableManagerV7, LeakFlagger {
   }
 
   void _throwOnInvalidCall(
-      String methodName, String parameterName, dynamic parameterValue) {
+      String methodName, String parameterName, Object parameterValue) {
```

Comment:
The analyzer doesn't like `dynamic` but it also doesn't like a missing type, so whatever, `Object` works just as well.